tests/test_virtualenv.py

```diff
@@ -332,3 +334,39 @@ def test_relative_symlink(tmpdir):
     shutil.rmtree(workdir)
 
     assert os.path.exists(lib64)
+
+
+def test_missing_certifi_pem(tmpdir):
+    """Make sure that we can still create virtual environment if pip is
+    patched to not use certifi's cacert.pem and the file is removed.
+    This can happen if pip is packaged by Linux distributions."""
+    tests_dir = os.path.dirname(__file__)
+    proj_dir = os.path.dirname(tests_dir)
+    support_original = os.path.join(proj_dir, "src", "virtualenv_support")
+    pip_wheel = sorted(glob.glob(os.path.join(support_original, "pip*whl")))[0]
+    whl_name = os.path.basename(pip_wheel)
+
+    wheeldir = tmpdir / "wheels"
+    wheeldir.mkdir()
+    tmpcert = tmpdir / "tmpcert.pem"
+    cacert = "pip/_vendor/certifi/cacert.pem"
```

Comment:
does not the same path separator counts on this and below (hence why still failing on Windows)?

Reply:
this is inside the zip but let me assert we found the file.

Reply:
asserts work, different problem